src/Products/ZCTextIndex/tests/testZCTextIndex.py

```diff
@@ -287,43 +290,43 @@ def test_z3interfaces(self):
         verifyClass(IZCTextIndex, ZCTextIndex)
 
     def testRanking(self):
-        self.words = ["cold", "days", "eat", "hot", "lot", "nine", "old",
-                      "pease", "porridge", "pot"]
-        self.docs = ["Pease porridge hot, pease porridge cold,",
-                     "Pease porridge in the pot,",
-                     "Nine days old.",
-                     "In the pot cold, in the pot hot,",
-                     "Pease porridge, pease porridge,",
-                     "Eat the lot."]
+        self.words = ['cold', 'days', 'eat', 'hot', 'lot', 'nine', 'old',
+                      'pease', 'porridge', 'pot']
+        self.docs = ['Pease porridge hot, pease porridge cold,',
+                     'Pease porridge in the pot,',
+                     'Nine days old.',
+                     'In the pot cold, in the pot hot,',
+                     'Pease porridge, pease porridge,',
+                     'Eat the lot.']
         self._ranking_index()
         self._ranking_tf()
         self._ranking_idf()
         self._ranking_queries()
 
         # A digression to exercise re-indexing.
         docs = self.docs
-        for variant in "hot cold porridge python", "pease hot pithy":
+        for variant in 'hot cold porridge python', 'pease hot pithy':
```

Comment:
Allthough that also work, I would prefer some parentheses like 
`('hot cold porridge python', 'pease hot pithy')`.